### PR TITLE
[Overlays] Twitch Env Variables and Chat Key

### PR DESCRIPTION
**Pull request by colbyfayock**
Fixes several issues with environment variables between a few typos and using the defined values throughout in spots where it makes sense to pull dynamically.

Also fixed a warning where the map function within the Twitch chat box would complain about not having a `key` prop. `index` isn't always ideal, but I couldn't find any sort of "id" associated with each message, and we're not doing any kind of rearranging within the chatbox for it to have a huge impact on rendering.

### CODE REVIEW

File: templates/mdnext-overlays/src/hooks/useChatListener.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default function useChatListener({ channel, commands = {} }) {
     },
     channels: [channel],
     identity: {
-      username: 'domitriusclark',
+      username: process.env.CHANNEL,
```
Comment (colbyfayock) [inline templates/mdnext-overlays/src/hooks/useChatListener.js:24]: will this always be the same as the channel property?

Comment (domitriusclark) [reply templates/mdnext-overlays/src/hooks/useChatListener.js:24]: yeah it should be

Comment (colbyfayock) [reply templates/mdnext-overlays/src/hooks/useChatListener.js]: cool i updated this so it just uses `channel` here

**Comment by domitriusclark** [comment]
Thanks again @colbyfayock 🙌

**Pull request merged**